orm/db_sqlite.go

```diff
@@ -66,6 +68,11 @@ type dbBaseSqlite struct {
 
 var _ dbBaser = new(dbBaseSqlite)
 
+// override base db read for update behavior as SQlite does not support syntax
+func (d *dbBaseSqlite) Read(q dbQuerier, mi *modelInfo, ind reflect.Value, tz *time.Location, cols []string, isForUpdate bool) error {
+	return d.dbBase.Read(q, mi, ind, tz, cols, false)
```

Comment:
Please log some warning information if the `isForUpdate` is `true`

Reply:
Hi @flycash . I have added the warn message to note user about isForUpdate param on SQLite is ignored and always set as false in calls. Please review. Thanks.